Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs

```diff
@@ -45,6 +55,13 @@ private static void OnAutoSizePropertyChanged(BindableObject bindable, object ol
         {
             multiLineInput.inputBorderlessEditor.AutoSize = multiLineInput.AutoSize ? (EditorAutoSizeOption)1 : (EditorAutoSizeOption)0;
         }
+    }
+    private static void OnIntValueChanged(BindableObject bindable, object oldValue, object newValue)
+    {
+        if (bindable is MultiLineInput multiLineInput && (int)newValue > 44)
```

Comment:
What happens when you change the value to 50 and then 40? Acccording to this logic, it will change into 50 first and then it will never do anything (but it should set ito to 44 when we try to set 40)

            `multiLineInput.inputBorderlessEditor.HeightRequest = Math.Max((int)newValue, 40);` should do the trick

Reply:
We are not in need to change the height request dynamically after setting it, hence after the height request is set before running the app , if it's less than 44 ,the value will not get set .